examples/features/xds/client/main.go

```diff
@@ -16,78 +16,56 @@
  *
  */
 
-// Package main implements a client for Greeter service.
+// Binary main implements a client for Greeter service using gRPC's client-side
+// support for xDS APIs.
 package main
 
 import (
 	"context"
 	"flag"
-	"fmt"
 	"log"
+	"strings"
 	"time"
 
 	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+	xdscreds "google.golang.org/grpc/credentials/xds"
 	pb "google.golang.org/grpc/examples/helloworld/helloworld"
 
 	_ "google.golang.org/grpc/xds" // To install the xds resolvers and balancers.
 )
 
-const (
-	defaultTarget = "localhost:50051"
-	defaultName   = "world"
+var (
+	target   = flag.String("target", "", "uri of the Greeter Server, e.g. 'xds:///helloworld-service:8080'")
+	name     = flag.String("name", "world", "name you wished to be greeted by the server")
+	xdsCreds = flag.Bool("xds_creds", false, "whether the server should use xDS APIs to receive security configuration")
```

Comment:
Make this a multi-choice, instead of a boolean?

Also, do the same for all languages?